alacritty/src/config/mod.rs

```diff
@@ -202,6 +202,15 @@ fn print_deprecation_warnings(config: &Config) {
             "Config persistent_logging is deprecated; please use debug.persistent_logging instead"
         );
     }
+
+    if config.scrolling.faux_multiplier().is_some() {
+        warn!(
+            target: LOG_TARGET_CONFIG,
+            "Config scrolling.faux_multiplier is deprecated; please use scrolling.multiplier for \
+             both usual and faux scrolling. You can enable/disable faux scrolling using \
+             `\\e[?1007h`/`\\e[?1007l` escape sequences respectively"
```

Comment:
This should be adjusted similarly to the comment about the change log entry. The `Removed` part is the only relevant thing here, we don't need to clarify what the alternate scroll escape sequence is.